aimon/assets/logs/aimon_tests.yaml

```diff
@@ -0,0 +1,123 @@
+          output_quality:
+            completeness: 0.875
+            conciseness: 0.9
+            context_classification: 0.4378
+            groundedness: 1
+            hallucination: 0.0322265625
+            instruction_adherence: 1
+            output_relevance: 1
+          request: "Contextual and general-knowledge Hallucination scores computed at the token, phrase, and sentence levels."
+          safety_metrics:
+            cbrn: 0.9983
+            code_detection: 0.9975
+            jailbreak: 0.9797
+            personal_harm: 0.9981
+            pii: 0.9964
+            prompt_injection: 0.9924
+            sql_prevention: 0.9994
+            toxicity: 0
+            unsafe_stereotypes: 0.9978
+          service: "aimon"
+          status: "info"
+          timestamp: 1752865171000
+          version: 0
+        message: "Response Evaluation"
+        service: "aimon"
+        tags:
+         - "source:aimon"
+         - "datadog.submission_auth:api_key"
+        timestamp: "2025-07-18T18:59:31Z"
+      id: "AwAAAZge5-Y4M7kiLQAAABhBWmdlNS1tTkFBQ2gyR2F0TDlPYWNBQUEAAAAkZjE5ODFlZTctZTYzOC00MmE2LTk2MWQtZjg3MWI1ZDY2YjRjAAAAAA"
+    message: |-
+      {
+        "id" : "AwAAAZge5-Y4M7kiLQAAABhBWmdlNS1tTkFBQ2gyR2F0TDlPYWNBQUEAAAAkZjE5ODFlZTctZTYzOC00MmE2LTk2MWQtZjg3MWI1ZDY2YjRjAAAAAA",
+        "content" : {
+          "service" : "aimon",
+          "attributes" : {
+            "output" : "Contextual and general-knowledge Hallucination scores computed at the token, phrase, and sentence levels.",
+            "request" : "Contextual and general-knowledge Hallucination scores computed at the token, phrase, and sentence levels.",
+            "company_id" : "ea58e3af-cb9c-4571-83fb-40b8c9d7803e",
+            "service" : "aimon",
+            "safety_metrics" : {
+              "toxicity" : 0,
+              "pii" : 0.9964,
+              "prompt_injection" : 0.9924,
+              "cbrn" : 0.9983,
+              "sql_prevention" : 0.9994,
+              "jailbreak" : 0.9797,
+              "unsafe_stereotypes" : 0.9978,
+              "personal_harm" : 0.9981,
+              "code_detection" : 0.9975
+            },
+            "version" : 0,
+            "application_id" : 889,
+            "output_quality" : {
+              "conciseness" : 0.9,
+              "context_classification" : 0.4378,
+              "groundedness" : 1,
+              "hallucination" : 0.0322265625,
+              "instruction_adherence" : 1,
+              "output_relevance" : 1,
+              "completeness" : 0.875
+            },
+            "status" : "info",
+            "timestamp" : 1752865171000
+          },
+          "message" : "Response Evaluation",
+          "timestamp" : "2025-07-18T18:59:31Z",
+          "tags" : [ "source:aimon", "datadog.submission_auth:api_key" ]
+        }
+      }
+    tags:
+     - "source:LOGS_SOURCE"
```

Comment:
@JoshPatel13 should this be replaced?

Reply:
I am unsure, I put it there cause thats what CI was returning. Should I ask logs why changing it to aimon causes the pipeline to break?